var/spack/repos/builtin/packages/r-biocgenerics/package.py

```diff
@@ -25,32 +25,16 @@
 from spack import *
 
 
-class RBiocgenerics(Package):
+class RBiocgenerics(RPackage):
```

Comment:
This was the only package that was even slightly different from the rest. I confirmed that the changes I made allow it to still install correctly.